src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml

```diff
@@ -0,0 +1,1110 @@
+name: nmdc_submission_schema
+description: Schema for creating Data Harmonizer interfaces for biosamples based on MIxS and other standards
+id: https://example.com/nmdc_submission_schema
+version: 0.0.0
+prefixes:
+  UO: http://purl.obolibrary.org/obo/UO_
+  qud: http://example.org/qud/
+  xsd: http://www.w3.org/2001/XMLSchema#
+  nmdc_sub_schema: https://example.com/nmdc_sub_schema/
+  linkml: https://w3id.org/linkml/
+  shex: http://www.w3.org/ns/shex#
+  schema: http://schema.org/
+default_prefix: nmdc_sub_schema
+default_range: string
+imports:
+  - linkml:types
+types:
+  unit:
+    name: unit
+    description: a string representation of a unit
+    notes:
+      - why isn't this picked up from the nmdc.yaml import?
+    mappings:
+      - qud:Unit
+      - UO:0000000
+    base: str
+    uri: xsd:string
+  decimal degree:
+    name: decimal degree
+    description: a float representation of a degree of rotation
+    base: float
+    uri: xsd:decimal
+  language code:
+    name: language code
+    description: a string representation of a language
+    base: str
+    uri: xsd:language
+enums:
+  BioticRelationshipEnum:
+    description: placeholder enum descr
```

Comment:
Would it make sense to remove all these placeholder descriptions with this PR?

Reply:
yes, I thought I had. Maybe that was in some different project.

Reply:
- https://github.com/microbiomedata/submission-schema/issues/239